src/components/MenuBars/__tests__/MenuBars.test.tsx

```diff
@@ -47,3 +62,54 @@ describe("MenuBars", () => {
     expect(container.getElementsByClassName("user-menu").length).toBe(1);
   });
 });
+
+describe("Mark me as Done Tooltip Logic", () => {
+  beforeEach(() => {
+    (useTimer as jest.Mock).mockReturnValue({timerExpired: false});
+  });
+
+  it("Does not expand the tooltip if the timer is not expired", () => {
+    (useTimer as jest.Mock).mockReturnValue({timerExpired: false});
+    const store = getTestStore({
+      participants: {
+        self: getTestParticipant(),
+        others: [],
+      },
+    });
+    const {asFragment} = render(createMenuBars(store));
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it("Does not expand the tooltip if the possibleVotes !== usedVotes", () => {
+    (useTimer as jest.Mock).mockReturnValue({timerExpired: false});
+    const store = getTestStore({
+      participants: {
+        self: getTestParticipant(),
+        others: [],
+      },
+    });
+    const {asFragment} = render(createMenuBars(store));
+    expect(asFragment()).toMatchSnapshot();
+  });
+  it("Does not expand the tooltip multiple times", () => {});
+
+  it("Does not expand the tooltip if the timer expired and user is already ready", () => {
+    jest.useFakeTimers();
+    (useTimer as jest.Mock).mockReturnValue({timerExpired: true});
+    const store = getTestStore({
+      participants: {
+        self: getTestParticipant({ready: true}),
+        others: [],
+      },
+    });
+    const {getByTestId} = render(createMenuBars(store));
+    const button = getByTestId("mark-as-done-button");
```

Comment:
Using test ids should be avoided if possible, since it doesn't resemble the real world usage of our application. It always better to test from a user perspective => Using text or labels to get elements.

Reply:
Replaced by using text now :)